ddtrace/sampler.py

```diff
@@ -313,7 +345,8 @@ def __init__(
         # Enforce sample rate constraints
         if not 0.0 <= sample_rate <= 1.0:
             raise ValueError(
-                "SamplingRule(sample_rate={!r}) must be greater than or equal to 0.0 and less than or equal to 1.0",
+                "SamplingRule(sample_rate={}) must be greater than or equal to 0.0 ".format(sample_rate)
+                + "and less than or equal to 1.0"
```

Comment:
```suggestion
                (
                    "SamplingRule(sample_rate={}) must be greater than or equal to "
                    "0.0 and less than or equal to 1.0"
                ).format(sample_rate)
```


you can break up a string onto multiple lines without needing `+`